index.html

```diff
@@ -4,32 +4,35 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../css/styles.css" />
```

Comment:
one "." should be good 👍 

```suggestion
    <link rel="stylesheet" href="./css/styles.css" />
```